org.bunkus.mkvtoolnix-gui.yaml

```diff
@@ -24,9 +24,8 @@ modules:
   - name: boost
     buildsystem: simple
     build-commands:
-      - ./bootstrap.sh --prefix=/app
-      - ./b2 -j $FLATPAK_BUILDER_N_JOBS headers
-      - cp -R boost /app/
+      - mkdir -p /app/include
+      - cp -R boost /app/include
```

Comment:
I don't understand why this is supposed to "make sure headers are cleaned up" (probably because I know next to nothing about Flatpak's build process). Can you please elaborate?

Additionally, are you sure that `b2 headers` is only copying the headers, not modifying them in any way (replacing version numbers or whatever)? I don't know anything about Boost's build system either.

Also, when storing the headers as `/app/include/boost/…`, the `--with-boost=/app` option to `configure` shouldn't be needed anymore (as `/app/include` should be searched for include files anyway).

Reply:
> I don't understand why this is supposed to "make sure headers are cleaned up" (probably because I know next to nothing about Flatpak's build process). Can you please elaborate?

See the `cleanup` property at the top of the manifest (`/app` prefix is omitted):
```
cleanup:
  - /include
  - /lib/cmake
  - /lib/pkgconfig
  - /share/man
  - '*.a'
  - '*.la'
```
This deletes `/app/include` at the end of the build. Your last commit installed the boost headers under `/app/boost`, which is obviously not on that list.

> Additionally, are you sure that b2 headers is only copying the headers, not modifying them in any way (replacing version numbers or whatever)? I don't know anything about Boost's build system either.

I checked this with `diff -ur`, the files are modified.

> Also, when storing the headers as /app/include/boost/…, the --with-boost=/app option to configure shouldn't be needed anymore (as /app/include should be searched for include files anyway).

This was not the case when I originally wrote the manifest, but it seems it has been fixed since then.

Reply:
Ah, right. Thanks!